lib/segment/src/segment.rs

```diff
@@ -1374,10 +1378,9 @@ impl SegmentEntry for Segment {
         // flush segment to capture latest state
         self.flush(true)?;
 
-        let tmp_path = self.current_path.join(format!("tmp-{}", Uuid::new_v4()));
-
-        let db_backup_path = tmp_path.join(DB_BACKUP_PATH);
-        let payload_index_db_backup_path = tmp_path.join(PAYLOAD_DB_BACKUP_PATH);
+        let temp_path = temp_path.join(format!("snapshot-{}", Uuid::new_v4()));
```

Comment:
why is `tmp-` replaced by `snapshot-` in the path?

Reply:
Because this adds context that it is used for (temporary) snapshot files. The `temp_path` itself before joining already identifies it as temporary directory.

Does that make sense?

Reply:
Makes sense :+1:

Reply:
:raised_hands: